azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/ServiceEndpoints.java

```diff
@@ -14,6 +14,9 @@
 public class ServiceEndpoints {
     private String aadSigninUri;
     private String aadGraphApiUri;
+    private String aadGraphApiUriNew;
     private String aadKeyDiscoveryUri;
     private String aadMembershipRestUri;
+    private String aadMembershipRestUriNew;
+    private String aadUseNewGraphApi;
```

Comment:
you can use `bool` directly, and the name `aadUseNewGraphApi` looks like action not true of false.